content/docs/2.14/operate/cluster.md

```diff
@@ -129,6 +129,15 @@ The Kubernetes client config used within KEDA Operator and KEDA Metrics Adapter
 | kube-api-burst      | cfg.Burst               | 30            | Set the burst for throttling requests sent to the apiserver    |
 | disable-compression | cfg.DisableCompression  | true          | Disable compression for response in k8s restAPI in client-go, see [this Kubernetes issue](https://github.com/kubernetes/kubernetes/issues/112296) for details |
 
+## Metrics Service Parameters
+
+The Metrics Service config used by the KEDA Metrics Adapter to connect to the KEDA Operator can be adjusted by passing the following command-line flags to the Adapter binary:
+
+|     Adapter Flag                    | Default Value                                     | Description                                    |
+| ----------------------------------- | ------------------------------------------------- | ---------------------------------------------- |
+| metrics-service-address             | keda-operator.keda.svc.cluster.local:9666         | The address of the GRPC Metrics Service Server |
+| metrics-service-grpc-authority      | ""                                                | Host Authority override for the Metrics Service if the Host Authority is not the same as the address used for the GRPC Metrics Service Server. |
```

Comment:
Could we add a note explaining where this can be useful? a brief description of your use case would be nice (from top level, but just for giving a real world case to improve the understanding of the parameter)

Reply:
let me know what you think of the updated comment